### DISCUSSION TITLE
Consolidating Fields of Matrix block after migration from 4 to 5 version

### DISCUSSION THREAD

**Discussion by mikolamin** [craftcms/cloud#29] [Q&A]
I see the  plan to introduce a console command to handle merging similar fields.

https://craftcms.com/docs/5.x/upgrade.html#consolidating-fields

Consolidating Fields
We plan to introduce a console command to handle merging similar fields. There is no harm in running your project with extra fields for a time after upgrading.

So, when can we expect this script to appear?

**Comment by mikolamin** [comment]
I've founded a duplicate  https://github.com/craftcms/cms/discussions/14554

**Comment by angrybrad** [comment]
I'm going to close this one out as it belongs on the cms repo, and you've found the relevant issue. Follow along there for status updates, please.
